tests/functional/partial_parsing/test_pp_vars.py

```diff
@@ -314,40 +314,33 @@ def dbt_profile_target(self):
         # calls 'load_config' before the tests are run.
         # Note: only the specified profile is rendered, so there's no
         # point it setting env_vars in non-used profiles.
-        os.environ["ENV_VAR_USER"] = "root"
-        os.environ["ENV_VAR_PASS"] = "password"
+        os.environ["ENV_VAR_HOST"] = "localhost"
         return {
             "type": "postgres",
             "threads": 4,
-            "host": "localhost",
```

Comment:
@gshank I think the adjust to the test here makes sense since user and pass are not in `connection_info`